docs/DeveloperGuide.adoc

```diff
@@ -297,57 +297,97 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |track a module I'm currently taking |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |untrack a module |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a module |locate details of modules that NUS provides
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |curious student |view a module's details |check out if I want to take the module
+
+|`* * *` |busy student |manage deadlines |keep track of my assignments and tasks
+
+|`* * *` |organized student |manage my grades |know roughly how well I'm doing in the module
+
+|`* *` |new student |schedule my modules |plan how to approach my future focus areas
+
+|`* *` |graduating student |check what modules I need to complete |
+
+|`* *` |computing student |plan my focus area modules |choose a specialization
+
+|`* *` |competitive student |calculate my GPA |
+
+|`* *` |busy student |prioritize certain modules (and their deadlines) |manage my time better
+
+|`* *` |organized student |manage files related to a module |add and tag files
+
+|`* *` |student |catalog past modules |access the content conveniently
+
+|`*` |student |generate a sample module planner |plan my modules
+
+|`*` |hardworking student |access past exam papers |
```

Comment:
The benefits for a function is subjective and may not be immediately obvious, so it would be safe to include the benefits for all user stories